### ISSUE TITLE
fluid.kdtree~ help file indicates wrong type of analysis

### ISSUE THREAD

**Issue by tedmoore** [flucoma/flucoma-max#289]
The text in the file says that the plot is spectral centroid and loudness, but it seems to me like it's the output of a dimensionality reduction algorithm probably UMAP, so I think the text should be changed to reflect that.
<img width="765" alt="Screenshot 2022-06-17 at 13 20 21" src="https://user-images.githubusercontent.com/1531144/174296931-9ab94c82-cb5e-45ad-8e50-e58881bdc4ae.png">

**Comment by jamesb93** [comment]
Good catch. This is because the text was grandfathered in from a time where it actually was that analysis but is, as you suspect, DR analysis.

**Comment by rconstanzo** [comment]
And can you have some `pointsizescale` mercy on this humungous dots.

**Comment by tedmoore** [comment]
😆

**Comment by jamesb93** [comment]
Solved in 764fea3 and 1b8e1da
